tests/functional/gpu_multi_container/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  drum:
+    image: docker.io/oleksandrromaniukdr/nim:drum_dev_env_8
```

Comment:
until we have a stable env and a better place. I need an env for functional tests.